docs/yul.rst

```diff
@@ -391,7 +391,7 @@ The following functions must be available:
 +---------------------------------------------+-----------------------------------------------------------------+
 | shru256(x:u256, y:u256) -> z:u256           | logical right shift of x by y                                   |
 +---------------------------------------------+-----------------------------------------------------------------+
-| sars256(x:s256, y:u256) -> z:u256           | arithmetic right shift of x by y                                |
+| saru256(x:u256, y:u256) -> z:u256           | arithmetic right shift of x by y                                |
```

Comment:
Are you sure about these? I think we've changed them.

Reply:
Argh!

Reply:
Changed.